src/modules/navigator/rtl.cpp

```diff
@@ -318,7 +318,13 @@ RTL::set_rtl_item()
 		}
 
 	case RTL_STATE_DESCEND: {
-			_mission_item.nav_cmd = NAV_CMD_WAYPOINT;
+			if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING) {
+				_mission_item.nav_cmd = NAV_CMD_LOITER_TO_ALT;
```

Comment:
@dagar For some reason this does not work for plain multirotors.

Reply:
Can you try to set a loiter radius? There is probably the same issue as in https://github.com/PX4/Firmware/pull/14742

Reply:
The `LOITER_TO_ALT` logic was only implemented for FW. 

https://github.com/PX4/Firmware/blob/d0dcbae4af7e0aa993d0fb48f9da0becc4a663ae/src/modules/navigator/mission_block.cpp#L236-L237